ios/ios_project_setup.md

```diff
@@ -0,0 +1,61 @@
+# How To Create A New Project
+## Create Xcode project and Github repo
+1. Create repo on Github with `.gitignore` and `README.md`. Usually, repos should be named with the format `<project-name>-ios` e.g `bose-stetson-ios`
+
+1. Create Xcode Project
+
+1. In the Xcode project directory, run:
+    ```
+    git init
+    git remote add origin <remote-url>
+    git pull origin master
+    git push --set-upstream origin master
+    ```
+    > Pull before pushing to get the Github-generated `.gitignore` so ignored files don't end up in the repo
+
+1. Update `README.md` to use [Intrepid README template](https://github.com/IntrepidPursuits/sherpa/blob/master/readme-template)
+1. Add and commit, then push to `master`
+
+## Add Cocoapods
+1. In project directory, run:
+    ```
+    pod init
+    pod install
+    ```
+    > Use `.xcworkspace` instead of `.xcodeproj` for this project from now on
+
+1. Decide whether Pods should be checked in or not. The automatically generated Podfile is set up to check them in. If they shouldn't be checked in, uncomment the `Pods` line in the Podfile.
+    > `# Pods/` -- Pods checked in
+    > `Pods/` -- Pods not checked in
+
+1. Add, commit and push Pod files
+
+## Jenkins
+
+1. In Xcode, click on the project target icon next to the play button, then click "Manage Schemes...". Click the checkbox to set your project scheme to `Shared`. This is required for Jenkins
+
+1. Add, commit and push the changes.
+
+1. Setup Github webhooks
+    - For pull requests, add:
```

Comment:
This should be handled automatically for you if you hit the "Manage hooks for you" within the Pull Request Builder plugin in your Jenkins project